src/style.css

```diff
@@ -1,3 +1,104 @@
-body {
+/*Reset default margins*/
+html,
+body,
+div,
+object,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+em,
+img,
+var,
+b,
+u,
+i,
+ol,
+ul,
+li,
+form,
+label,
+legend,
+table,
+tr,
+th,
+td,
+footer,
+header,
+menu,
+nav,
+section {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-active-color: #00acdc;
+  --link-color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: auto;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  background-color: #fff;
+  margin-right: -20px;
+  list-style-type: none;
+  height: 100%;
+  flex-wrap: wrap;
+}
+
+.nav__link {
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  text-decoration: none;
+  color: var(--link-color);
+  font-family: "Roboto", sans-serif;
```

Comment:
3'rd point in checklist

Reply:
3'rd point states "Remember to use fallback fonts".
All I could find for 'Robot' font is 'sans-serif'🤷‍♂️

Could you please suggest what exactly do you refer to?